lib/actions/deploy.coffee

```diff
@@ -133,7 +133,7 @@ module.exports =
 
 				# Setup the build args for how the build routine expects them
 				options = _.assign({}, options, { appName })
-				params = _.assign({}, params, { context })
```

Comment:
There's still a `context` variable available from
```coffee
 		parseInput(params, options)
 		.then ([appName, build, context, imageName]) ->
```
I'm guessing that should be dealt with as part of this PR?